emulator/src/main/java/org/wso2/carbon/protocol/emulator/example/http/SimpleEmulatorServer.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * *
+ *  * Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *  *
+ *  * WSO2 Inc. licenses this file to you under the Apache License,
+ *  * Version 2.0 (the "License"); you may not use this file except
+ *  * in compliance with the License.
+ *  * You may obtain a copy of the License at
+ *  *
+ *  * http://www.apache.org/licenses/LICENSE-2.0
+ *  *
+ *  * Unless required by applicable law or agreed to in writing,
+ *  * software distributed under the License is distributed on an
+ *  * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  * KIND, either express or implied.  See the License for the
+ *  * specific language governing permissions and limitations
+ *  * under the License.
+ *
+ */
+package org.wso2.carbon.protocol.emulator.example.http;
+
+import io.netty.handler.codec.http.HttpMethod;
+import io.netty.handler.codec.http.HttpResponseStatus;
+import org.wso2.carbon.protocol.emulator.dsl.Emulator;
+
+import static org.wso2.carbon.protocol.emulator.http.server.contexts.HttpServerConfigBuilderContext.configure;
+import static org.wso2.carbon.protocol.emulator.http.server.contexts.HttpServerRequestBuilderContext.request;
+import static org.wso2.carbon.protocol.emulator.http.server.contexts.HttpServerResponseBuilderContext.response;
+
+/**
+ * Simple emulator server
+ */
+public class SimpleEmulatorServer {
+
+    public static void main(String args[]) {
+
+        Emulator.getHttpEmulator().server().given(configure().host("127.0.0.1").port(6065).context("/users"))
```

Comment:
These variables are used only for creating sample emulator server and client. They are not used in other places.